src/sentry/tasks/weekly_reports.py

```diff
@@ -99,7 +99,7 @@ def __repr__(self):
     max_retries=5,
     acks_late=True,
 )
-def schedule_organizations(dry_run=False, timestamp=None, duration=None):
+def schedule_organizations(dry_run=False, timestamp=None, duration=None, skip_flag_check=False):
```

Comment:
@Neo-Zhixing I think something like `use_new_format` or something is a better name for the argument

Reply:
If you’re calling this function you’re already using the new format.
This is just skipping the flag check.

Reply:
Correct. This is a different module.